opendebates/tests/test_submission.py

```diff
@@ -97,6 +99,38 @@ def test_post_submission(self):
         self.assertEqual(submission.approved, True)
         # vote count is kept on the submission model
         self.assertEqual(submission.votes, 1)
+        self.assertEqual(submission.current_votes, 1)
+        # but Vote objects are also created
+        votes = Vote.objects.filter(voter=self.voter)
+        self.assertEqual(1, len(votes))
+
+    def test_post_submission_after_previous_debate(self):
+        mode, _ = SiteMode.objects.get_or_create()
+        mode.previous_debate_time = timezone.make_aware(
+            datetime.datetime(2016, 1, 1, 12)
```

Comment:
Good call.  I've updated all of the tests where I did this to use `timezone.now() - datetime.timedelta(days=7)`.